script.js

```diff
@@ -68,12 +70,49 @@ textInput.addEventListener('keyup', event => {
                 top: scrollPixels += lineHeight,
                 behavior: 'smooth'
             })
-        
+
         }
         upcomingWord.style.color = upcomingWordColour
 
+
+        const wordsPerMinuteResult = document.querySelector('#wordsPerMinuteResult')
+        const accuracyResult = document.querySelector('#accuracyResult')
+
+        let totalAttemptedWords = wordIndexCount + 1
+        let wordsPerMinute = numberOfValidWords
+        let accuracy = Math.round((wordsPerMinute / totalAttemptedWords) * 100) + '%'
+        wordsPerMinuteResult.innerHTML = wordsPerMinute
+        accuracyResult.innerHTML = accuracy
```

Comment:
thinking about this, why are you doing it on every word? this only needs to be calculated when the time is up at the end

Reply:
Done: we calculate the accuracy and print the results in the HTML page only when the time is up